axon-server/installation/docker-k8s/axon-server-se.md

```diff
@@ -0,0 +1,278 @@
+# Axon Server SE
+
+This section is split into 3 sub-sections.
+
+* [Using the ready to use Axon Server SE Docker Image](axon-server-se.md#docker-image)
+* [Deployment using Docker Compose](axon-server-se.md#docker-compose)
+* [Deployment using Kubernetes](axon-server-se.md#kubernetes)
+
+## Docker Image
+
+Axon provides a ready to use [Axon Server SE image](https://hub.docker.com/r/axoniq/axonserver/). The image is built using a compact image from Google’s “distroless” base images at the gcr.io repository, in this case “gcr.io/distroless/java:11”.
+
+To run the provided image the command below can be executed. This starts Axon Server SE in a docker container with exposes the HTTP \(8024\) and GRPC \(8124\) ports to the host.
+
+```bash
+$ docker run -d --name  -p 8024:8024 -p 8124:8124 axoniq/axonserver
+```
+
+A quick verification of the running docker container can be done by querying the REST API that is available to retrieve configuration information for a running Axon Server instance.
+
+```bash
+$ curl -s http://localhost:8024/v1/public/me
+```
+
+This displays the information below
+
+```bash
+{
+  "authentication": false,
+  "clustered": false,
+  "ssl": false,
+  "adminNode": true,
+  "developmentMode": false,
+  "storageContextNames": [
+    "default"
+  ],
+  "contextNames": [
+    "default"
+  ],
+  "httpPort": 8024,
+  "grpcPort": 8124,
+  "internalHostName": null,
+  "grpcInternalPort": 0,
+  "name": "{axon-server-name}",
+  "hostName": "{axon-server-hostname}"
+}
+```
+
+The application is installed in the root with a minimal properties file which is depicted below. The “/data” and “/eventdata” directories are created as volumes, and their data will be accessible on the local filesystem somewhere in Docker’s temporary storage tree.axoniq.axonserver.event.storage=/eventdata
+
+```text
+axoniq.axonserver.snapshot.storage=/eventdata
+axoniq.axonserver.controldb-path=/data
+axoniq.axonserver.pid-file-location=/data
+logging.file=/data/axonserver.log
+logging.file.max-history=10
+logging.file.max-size=10MB
+```
+
+### Customization
+
+The directory locations for the volumes can be specified as per your requirements. The image also has a third directory "/config" which is not marked as a volume. This gives you the capability to have an "axonserver.properties" file which can be placed in this location to override the above mentioned settings as well as add new properties similar to a local install.
+
+Assuming that you have a directory "_axonserverse**"**_ which will be the designated location for your volumes and configuration information.
```

Comment:
What is the purpose of the double-star notation?

Reply:
Italics